src/videos.md

```diff
@@ -61,6 +70,16 @@ Alvaro Sánchez Villar of the Princeton Plasma Physics Laboratory presented "MFE
 
 ---
 
+#### Brian Young
+#### *OpenParEM2D: A 2D Simulator for Guided Waves*
+##### **October 25, 2022** | [MFEM Workshop 2022](workshop.md)
+
+![YouTube](_CqFJUu7E8E)
+
+Independent software developer Brian Young presented "OpenParEM2D: A Free, Open-Source Electromagnetic Simulator for 2D Waveguides and Transmission Lines." An overview is provided on a 2D electromagnetic simulator for guided waves called OpenParEM2D. It is an open-source and free project licensed under GPLv3 or later and released at the website https://openparem.org. Capabilities and methodology are presented.
```

Comment:
`https://openparem.org` doesn't show up as a link for me. Maybe replace by

- `at the website [https://openparem.org](https://openparem.org)`
 
or

- `at its [website](https://openparem.org)`